### PR TITLE
[PPhuman application] application of action model training in PPhuman 

### PR DESCRIPTION
**Pull request by zoooo0820**
What this pr did:
- add configs / script / document for training and exporting action model used in PPHuman.
- support COCO format keypoint for model STGCN
- add training & test support when class < 5.

### CODE REVIEW

File: applications/PPHuman/README.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+├── det_keypoint_unite_image_results_fall-01-cam0-rgb.json
+├── det_keypoint_unite_image_results_fall-02-cam0-rgb.json
+├── det_keypoint_unite_image_results_fall-03-cam0-rgb.json
+├── det_keypoint_unite_image_results_fall-04-cam0-rgb.json
+    ...
+├── det_keypoint_unite_image_results_fall-28-cam0-rgb.json
+├── det_keypoint_unite_image_results_fall-29-cam0-rgb.json
+└── det_keypoint_unite_image_results_fall-30-cam0-rgb.json
+```
+这里使用我们提供的脚本直接将数据转化为训练数据, 得到数据文件`train_data_ur_fall_50.npy`, 标签文件`train_label_ur_fall_50.pkl`。该脚本执行的内容包括解析json文件内容、前述步骤中介绍的整理训练数据及保存数据文件。
```
Comment (jerrywgz) [inline applications/PPHuman/README.md]: 这个脚本的输出改成相对通用的名称吧

Comment (zoooo0820) [reply applications/PPHuman/README.md]: Done

**Review by jerrywgz — APPROVED** [review]

File: applications/PPHuman/README.md [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+
+在训练完成后，采用以下命令进行预测：
+```bash
+python main.py --test -c applications/PPHuman/configs/stgcn_pphuman.yaml  -w output/STGCN/STGCN_best.pdparams
+```
+
+### 导出模型推理
+
+- 在PaddleVideo中，通过以下命令实现模型的导出，得到模型结构文件`STGCN.pdmodel`和模型权重文件`STGCN.pdiparams`，并增加配置文件：
+```python
```
Comment (huangjun12) [inline applications/PPHuman/README.md]: python --> bash

File: applications/PPHuman/README.md [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+# current path is under root of PaddleVideo
+python tools/export_model.py -c applications/PPHuman/configs/stgcn_pphuman.yaml \
+                                -p output/STGCN/STGCN_best.pdparams \
+                                -o output_inference/STGCN
+
+cp applications/PPHuman/configs/infer_cfg.yml output_inference/STGCN
+
+# 为模型增加软链接
+cd output_inference/STGCN
+ln -s STGCN.pdiparams model.pdiparams
```
Comment (huangjun12) [inline applications/PPHuman/README.md]: 这里为什么用软链接，不直接rename呢

File: applications/PPHuman/README.md [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+# current path is under root of PaddleDetection
+
+# Step 1: download pretrained inference models.
+wget https://bj.bcebos.com/v1/paddledet/models/pipeline/mot_ppyoloe_l_36e_pipeline.zip
+wget https://bj.bcebos.com/v1/paddledet/models/pipeline/dark_hrnet_w32_256x192.zip
+unzip -d output_inference/ mot_ppyoloe_l_36e_pipeline.zip
+unzip -d output_inference/ dark_hrnet_w32_256x192.zip
+
+# Step 2: Get the keypoint coordinarys
+
```
Comment (huangjun12) [inline applications/PPHuman/README.md:58]: 加一下注释？

\# if your data is image sequence

**Comment by huangjun12** [comment]
首页文档 Application下也加一下这个链接吧

File: applications/PPHuman/README.md (hunk 2 above)
Comment (zoooo0820) [reply applications/PPHuman/README.md]: Done, thanks

File: applications/PPHuman/README.md (hunk 3 above)
Comment (zoooo0820) [reply applications/PPHuman/README.md]: 已修改

File: applications/PPHuman/README.md (hunk 4 above)
Comment (zoooo0820) [reply applications/PPHuman/README.md:58]: Done, thanks

**Pull request merged**